src/main/kotlin/it/pagopa/wallet/config/properties/CdcQueueConfig.kt

```diff
@@ -2,10 +2,10 @@ package it.pagopa.wallet.config.properties
 
 import org.springframework.boot.context.properties.ConfigurationProperties
 
-@ConfigurationProperties("cdc-queue")
+@ConfigurationProperties("cdc.queue")
 data class CdcQueueConfig(
     val storageConnectionString: String,
     val storageQueueName: String,
     val ttlSeconds: Long,
-    val timeoutWalletExpired: Long
+    val timeoutWalletCdc: Long
```

Comment:
right name is hard work :-) can you specify the name of this variable? what does cdc wallet timeout mean? maybe it refers to the delay with which the message will be read by the consumer? why provide this delay?

Reply:
the variable refers the visibilityTimeout supported by the azure queue 
i renamed it to visibilityTimeoutWalletCdc to make it more readable.
the value comes from the ENV variable CDC_QUEUE_VISIBILITY_TIMEOUT_SECONDS wich is set to 0
i decided to keep the parameter just in case in the future we need to eventually tune this.

fixed here ec7d7146cf5c1b1f5100c1723a7fbcd4b876d82f